### PR TITLE
Fix apt-key references, links in global parameters, minimum requirements and version locking documentation

### LINKED ISSUE #270: apt-key deprecation in Debian 11 and Ubuntu 22.04
### Remove deprecated *apt-key* command
Update installation process, documentation and/or repository to use best-practice replacement for the soon to be deprecated *apt-key* command currently used.

### Use of signed-by attribute in sources.list.d repo entry recomended (https://wiki.debian.org/DebianRepository/UseThirdParty)
* This is an alternate method supported since atleast apt version 1.1exp15
* Debian 9+ (stretch) supports apt version 1.4.11 (https://packages.debian.org/stretch/apt)
* Support for this feature may be lacking in Debian 8 (jessie) and older.
* Placing the repository key in /usr/share/keyrings/gravwell-archive-keyring.gpg and using the signed-by attribute appears to be a best practice and secure alternative to using *apt-key*
* Have not fully researched impact to Ubuntu or other *nix

### Current Process
```
sudo apt install apt-transport-https gnupg curl
curl https://update.gravwell.io/debian/update.gravwell.io.gpg.key | sudo apt-key add -
echo 'deb [ arch=amd64 ] https://update.gravwell.io/debian/ community main' | sudo tee /etc/apt/sources.list.d/gravwell.list
```

### Recomended Process
```
sudo apt install apt-transport-https
sudo wget -O /usr/share/keyrings/gravwell-archive-keyring.asc https://update.gravwell.io/debian/update.gravwell.io.gpg.key
sudo echo 'deb [ arch=amd64 signed-by=/usr/share/keyrings/gravwell-archive-keyring.asc ] https://update.gravwell.io/debian/ community main' | sudo tee /etc/apt/sources.list.d/gravwell.list
```

Make sure to call out the old method and describe which versions should use the old method.

### LINKED ISSUE #288: Document component and host compatibility
We need a compatibility page that discusses the following:

* version locking on searchagent, indexer, and webserver
* Host compatibility - linux 3.2+ and SystemD
* Ingester compatibility - Discuss version locking and major release compatibility garuntee

### LINKED ISSUE #308: Make global parameter list linkable
For example, i need to be able to link to the Global parameter "Webserver-Ingest-Groups" on this page: 
https://docs.gravwell.io/#!configuration/parameters.md#Global_Parameters

### PR DESCRIPTION
**Pull request by djfritz**
Fixes #270 
Fixes #288 
Fixes #308

### CODE REVIEW

File: architecture/architecture.md [hunk 1]
```diff
@@ -37,3 +37,13 @@ The Gravwell ingest API is designed to be fast and simple.  The API and many of
 The core ingest mechanic requires only three data items: a byte array, timestamp, and a tag (which is provided by the API).  Simple installations might have a few ingesters talking directly to indexers where more complicated ingesters might have multiple levels of federation as data is shuffled up from untrusted enclaves to high security enclaves.  Visibility is key to security, analytics, and hunt operations.  Segregating data or forcing multiple installations where operators must manually correlate data is not acceptable.  Gravwell achieves federation via ingest relay ingesters, which speak the ingest API on both sides and allow dual homed machines to aggregate and relay data.
 
 ![An example of a complex ingest structure](IngestRelay.png)
+
+## Compatibility 
+
+### Minimum Software Requirements
+
+Gravwell runs on most common Linux distributions. A minimum kernel version of 3.2 and SystemD are required.
```
Comment (traetox) [inline architecture/architecture.md]: call out AMD64 here too.

Comment (traetox) [reply architecture/architecture.md]: PR landed

**Review by traetox — APPROVED** [review]
LGTM

File: architecture/architecture.md (hunk 1 above)
Comment (djfritz) [reply architecture/architecture.md]: LGTM

**Pull request merged**